docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,262 @@
+   terminate the test.
+
+10. If the *SPF Policy* contains more than one `redirect` modifier, or more than
+    one `exp` modifier, then output *[Z11_SPF1_MULTIPLE_SAME_MODIFIER]*.
+
+11. If the *SPF Policy* contains exactly one `redirect` modifier anywhere except
+    at the end of the policy text, then output *[Z11_SPF1_REDIRECT_NOT_AT_END]*.
+
+12. If the *SPF Policy* contains both a `redirect` modifier and an `all`
+    mechanism, then output *[Z11_SPF1_REDIRECT_AND_ALL]*.
+
+13. If the *SPF Policy* contains at least one `ptr` mechanism, then output
+    *[Z11_SPF1_HAS_PTR]*.
+
+14. Count ("DNS Lookup count") the number of terms in the *SPF Policy* that
+    are either:
+
+   1. A mechanism of type `a`, `mx`, `ptr`, `exists`;
+
+   2. Or a modifier of type `redirect`.
+
+15. If the `%{p}` [macro] appears as a domain-spec for an [SPF term] in the
+    *SPF Policy*, then:
+
+   1. Output *[Z11_SPF1_HAS_P_MACRO]*.
+
+   2. If the same policy contains no `ptr` mechanism, then increment the *DNS
+   Lookup count*.
+
+16. If the *DNS Lookup count* is greater than 10, then output
+    *[Z11_SPF1_TOO_COMPLEX]*.
+
+17. If the *DNS Lookup count* is less than or equal to 10 and the *SPF Policy*
+    contains a `redirect` modifier or at least one `include` mechanism, then
+    output *[Z11_SPF1_RECURSIVE]*.
+
+18. If no other message was outputted by this test case, then output
+    *[Z11_SPF1_SYNTAX_OK]*.
+
+## Outcome(s)
+
+The outcome of this Test Case is "fail" if there is at least one message
+with the severity level *[ERROR]* or *[CRITICAL]*.
+
+The outcome of this Test Case is "warning" if there is at least one message
+with the severity level *[WARNING]*, but no message with severity level
+*ERROR* or *CRITICAL*.
+
+In other cases, no message or only messages with severity level
+*[INFO]* or *[NOTICE]*, the outcome of this Test Case is "pass".
+
+## Special procedural requirements
+
+If either IPv4 or IPv6 transport is disabled, skip sending queries over that
+transport protocol. A message will be outputted reporting that the transport
+protocol has been skipped.
+
+## Intercase dependencies
+
+None.
+
+## Terminology
+
+When the term "concatenate" is used, it refers to the conversion of a TXT
+resource record’s data to a single contiguous string, as specified in [RFC 7208,
+§ 3.3][RFC7208#3.3].
+
+When the term "macro" is used, it refers to the macros permissible in SPF
+domain-specs, as specified in [RFC 7208, § 7][RFC7208#7].
+
+When the term "passing the syntax check" is used, it means that the text is
+valid according to the ABNF grammar published in [RFC 7208][RFC7208] starting
+from [§ 4.5][RFC7208#4.5].
+
+When the term "using Method" is used, names and IP addresses are fetched
+using the defined [Methods].
+
+When the term "SPF term" is used, it includes both SPF mechanisms as defined in
+[RFC 7208, § 5][RFC7208#5] and modifiers as defined in [RFC 7208, §
+6][RFC7208#6].
```

Comment:
For this test case specification there are five terms. This is the format used in other specifications, but I wonder if it would be more readable if the term is topicalized and using a format like the following:

* "concatenate" - When the term is used, it refers to the conversion of a TXT
resource record’s data to a single contiguous string, as specified in [RFC 7208,
§ 3.3][RFC7208#3.3].
* "macro" - When the term is used, it refers to the macros permissible in SPF
domain-specs, as specified in [RFC 7208, § 7][RFC7208#7].

Reply:
Works for me. In that case we should consider updating the template and the specifications that already follow the new template.

Reply:
This PR is merged, so I’ll rebase the next iteration on the new `develop` branch and rewrite the test cases so that they follow the updated template.